contracts/0.8.9/oracle/BaseOracle.sol

```diff
@@ -314,7 +314,7 @@ abstract contract BaseOracle is IReportAsyncProcessor, AccessControlEnumerable,
         if (addr == address(0)) revert AddressCannotBeZero();
 
         address prevAddr = CONSENSUS_CONTRACT_POSITION.getStorageAddress();
-        if (addr == prevAddr) revert AddressCannotBeSame();
+        //if (addr == prevAddr) revert AddressCannotBeSame();  // Certora munging: commented out
```

Comment:
Can you explain?

Reply:
This is for the AccountingOracle verification. There I am forcing the address of the Consensus contract to be the one I explicitly add to the scene (using require). And because of that requirement, any time a call to the function that changes the address of the consensus contract is done, I get reachability issues.